### PR TITLE
Update HACKING.md

### PR DESCRIPTION
**Pull request by notriddle**
Match up with some reorganization that has been done in the meantime.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -14,33 +14,26 @@ You'll also want to do some terminology mapping between what we say to users and
...
-In `bors_frontend/lib/bors_ng`, the worker GenServers are implemented.
-This is also where Phoenix's Web and Endpoint modules are
-(even though they're technically part of the web frontend, they can't go in
-`web/` because they're part of Phoenix's hot reload setup).
+In `bors_worker`, the worker GenServers are implemented.
 
-### Batcher and Attemptor
+### `Worker.Batcher` and `Worker.Attemptor`
 
 Batcher handles "r+"-ed patches, which the user is attempting to land in master.
```
Comment (Macarse) [inline HACKING.md]: > Batcher handles

 We need to find a common word for the interface/protocol which represents: 
 - The `Batcher`
 - The `Attemptor`
 - The `Parraleler` (https://github.com/bors-ng/bors-ng/issues/141)

File: HACKING.md [hunk 2]
```diff
@@ -14,33 +14,26 @@ You'll also want to do some terminology mapping between what we say to users and
   * Attempt → Try: Try is a keyword in Elixir, and in a lot of other languages besides,
                    so the code calls a try an attempt.
   * Patch → PR: A Bors "Patch" corresponds to a GitHub "Pull Request."
-  * Aelita2 → Bors-NG: I renamed it without changing the name in the code.
```
Comment (Macarse) [inline HACKING.md]: I didn't remove this line in my changes to this file since "aelita" is still used in some instances of the repo.

**Comment by Macarse** [comment]
👍

File: HACKING.md (hunk 1 above)
Comment (notriddle) [inline HACKING.md]: Worker?

Comment (Macarse) [inline HACKING.md]: After I threw `The Parraleler` you asked me for a name??? 🤣 
I think Worker for now would be fine but the work they do is more of a `Scheduler`.

Comment (notriddle) [inline HACKING.md]: "Worker" seems more general than "Scheduler" (which is one of the reasons Syncer is in there).

So, next step in this refactoring: merge the Batcher and Attemptor registries into a single SchedulerRegistry, so Batcher and Attemptor will be known as Schedulers.

Comment (notriddle) [inline HACKING.md]: And, yeah, I'm just figuring between the both of us, we'll be able to come up with a name that doesn't sound stupid.

**Comment by notriddle** [comment]
bors r=Macarse

**Pull request merged**